libcontainerd/client_windows.go

```diff
@@ -362,7 +362,19 @@ func (clnt *client) Signal(containerID string, sig int) error {
 		}
 
 		// Shutdown the compute system
-		if err := hcsshim.ShutdownComputeSystem(containerID, hcsshim.TimeoutInfinite, context); err != nil {
+		const shutdownTimeout = 5 * 60 * 1000 // 5 minutes
+		if err := hcsshim.ShutdownComputeSystem(containerID, shutdownTimeout, context); err != nil {
+			if herr, ok := err.(*hcsshim.HcsError); !ok ||
+				(herr.Err != hcsshim.ERROR_SHUTDOWN_IN_PROGRESS &&
+					herr.Err != ErrorBadPathname &&
+					herr.Err != syscall.ERROR_PATH_NOT_FOUND) {
+				logrus.Warnf("signal - error from ShutdownComputeSystem %v. Calling TerminateComputeSystem", err)
+				if err := hcsshim.TerminateComputeSystem(containerID, shutdownTimeout, "signal"); err != nil {
+					logrus.Warnf("signal - ignoring error from TerminateComputeSystem %v", err)
```

Comment:
Will be helpful to make sure the container ID is included here.
Also is this a real warning or can it be debug?